clients/vscode-hlasmplugin/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 - Incorrect processing of self-referencing data definition statements
 - Enhanced multiline support in TextMate grammar for source files
 - Better handling of Unicode Supplementary Planes
+- Correct processor group now assigned to programs with special characters stated in `.bridge.json`
```

Comment:
This seems a bit convoluted? Not sure how to rephrase it though...

Reply:
What about one of these?

- Incorrect processor group could be assigned to programs stated in `.bridge.json`
- Improved pairing of processor groups to programs stated in `.bridge.json`

Reply:
```
Incorrect processor group could be assigned to programs stated in .bridge.json
```
sounds good